src/phantom.cpp

```diff
@@ -110,7 +110,7 @@ void Phantom::init()
         QWebSettings::setOfflineStorageDefaultQuota(m_config.offlineStorageDefaultQuota());
     }
 
-    m_page = new WebPage(this, QUrl::fromLocalFile(m_config.scriptFile()));
+    m_page = new WebPage(this, QUrl(QString(JAVASCRIPT_SOURCE_CODE_URL).arg("html")));
```

Comment:
This is not crucial for fixing debugger so I can revert it back to what it originally was.
I put this change so that we could have nicer and more sense-able arrangement of source files.